src/api/sso/test/e2e/row-level-policies.test.js

```diff
@@ -0,0 +1,350 @@
+
+        expect(status).toBe(200);
+        expect(count).toBe(feedList.length);
+      });
+
+      it('should not allow adding new feeds', async () => {
+        const { status } = await supabase.from('feeds').insert(
+          {
+            url: 'http://dev.to/devto',
+            wiki_author_name: 'devto',
+          },
+          { returning: 'minimal' }
+        );
+
+        // forbidden, unauthenticated users can't add new feeds
+        expect(status).toBe(403);
+      });
+
+      it('should not allow updating feeds', async () => {
+        const { status } = await supabase
+          .from('feeds')
+          .update({ html_url: 'http://dev.to/' }, { returning: 'minimal' })
+          .match({ url: feedList[0].url });
+
+        expect(status).toBe(404);
+      });
+
+      it('should not allow deleting feeds', async () => {
+        const { status, count } = await supabase.from('feeds').delete({ count: 'exact' }).match({
+          url: feedList[0].url,
+        });
+
+        // No feeds are deleted
+        expect(status).toBe(200);
+        expect(count).toBe(0);
+      });
+    });
+  });
+
+  describe('Authenticated users through Seneca SSO', () => {
+    const { email, firstName, lastName, displayName, githubAvatarUrl } = telescopeUser1;
+    const accessToken = createToken(
+      email,
+      firstName,
+      lastName,
+      displayName,
+      ['seneca', 'telescope'],
+      githubAvatarUrl
+    );
+    const supabase = createClient(SUPABASE_URL, ANON_KEY);
+    // Override the JWT on the public client
+    supabase.auth.setAuth(accessToken);
+
+    describe('"telescope_profiles" security policies', () => {
+      beforeEach(() => createTelescopeUsers(users));
+
+      afterEach(() => cleanupTelescopeUsers(users));
+
+      it('should return all users', async () => {
+        const { status, count } = await supabase
+          .from('telescope_profiles')
+          .select('id', { count: 'exact' });
+
+        expect(status).toBe(200);
+        expect(count).toBe(users.length);
+      });
+
+      it('should not allow creating new users', async () => {
+        const { status } = await supabase.from('telescope_profiles').insert(
+          {
+            id: hash(unregisteredUser.email),
+            email: unregisteredUser.email,
+            first_name: unregisteredUser.firstName,
+            last_name: unregisteredUser.lastName,
+            display_name: unregisteredUser.displayName,
+            github_username: unregisteredUser.githubUsername,
+            github_avatar_url: unregisteredUser.githubAvatarUrl,
+          },
+          { returning: 'minimal' }
+        );
+
+        expect(status).toBe(401);
+      });
+
+      it('should allow users to update their own profiles', async () => {
+        const { status, data: profiles } = await supabase
+          .from('telescope_profiles')
+          .update({ display_name: unregisteredUser.displayName }, { returning: 'representation' })
```

Comment:
To make this less confusing, I'd avoid using `unregisteredUser` here, since you're not really doing anything with that user.  All you need is a new string:

```js
const newDisplayName = 'Updated';

.update({ display_name: newDisplayName}, { returning: 'representation' })
...
expect(profiles[0].display_name).toEqual(newDisplayName);
```